app/views/buyers/accounts/_menu.html.slim

```diff
@@ -7,7 +7,7 @@ ul#subsubmenu
         class: "SubSubMenuItem-link"
 
   li.SubSubMenuItem.SubSubMenuItem--branch
-    = link_to_unless_current_styled pluralize(number_to_human(@account.bought_cinstances.count), 'Application'),
+    = link_to_unless_current_styled pluralize(number_to_human(current_user.accessible_cinstances.count), 'Application'),
```

Comment:
@thomasmaas This doesn't scope the applications by the developer account, does it? It's only the logged-in admin user.